citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/CitationManager.java

```diff
@@ -74,6 +76,9 @@
     
     @Autowired
     private ICitationDao citationDao;
+    
+    @Autowired
+    private ICitationVersionsDao citationVersionsDao;
```

Comment:
is this in the model plugin? Let's keep it in citesphere itself for now, since all the other javers stuff is in citesphere too.